File: src/main/java/com/example/fileservice/library/Logger.java
package com.example.fileservice.library;

import org.springframework.stereotype.Component;

@Component
public class Logger {

    public void crit(LogItem logItem) {
        write(logItem);
    }

    private void write(LogItem logItem) {
        System.out.println("crit" + ": " + logItem);
    }
}

File: src/main/kotlin/com/example/fileservice/library/log/Logger.kt
package com.example.fileservice.library.log

import org.springframework.stereotype.Component

@Component
class Logger {

    fun crit(logItem: LogItem) {
        write("crit", logItem)
    }

    private fun write(logLevel: String, logItem: LogItem) {
        println("$logLevel: $logItem")
    }
}
